CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### v0.9.6 - 26 Sept 2021
+Update users example with input examples
+

```

Comment:
CHANGELOG is not updated now... 😢

Reply:
If we update it halfway, users may be confused, so let us only use the release notes for now.

Reply:
related: #987